### PR TITLE
Frmw626 move Components/tooltips to Elements/Tooltips

### PR DESCRIPTION
**Pull request by kevincareer427**
JIRA: https://jira.rax.io/browse/FRMW-626

### LGTMs
- [x] Dev LGTM
- [x] Dev LGTM

### CODE REVIEW

**Comment by CITguy** [comment]
@kevincareer427: It looks like tests are failing because of a change in the rxDatePicker page object that's not in your code. Try rebasing your work on 2.x to see if that helps.

**Comment by kevincareer427** [comment]
@CITguy already rebased

File: utils/rx-page-objects/src/rxDatePicker.page.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ class rxDatePicker {
      * **Format:** `YYYY` (e.g. "2016")
      */
     get year () {
-        return rxSelect.initialize(this.rootElement.element(by.model('currentYear'))).selectedOption.text;
```
Comment (CITguy) [inline utils/rx-page-objects/src/rxDatePicker.page.js]: Good catch!

**Comment by 100stacks** [comment]
@kevincareer427, can you add the new demo page and API screenshots?  Thanks!

**Comment by kevincareer427** [comment]
## Before
before

## After
after

**Comment by kevincareer427** [comment]
@100stacks Added, thanks!

**Comment by 100stacks** [comment]
LGTM

**Comment by CITguy** [comment]
LGTM

**Pull request merged**